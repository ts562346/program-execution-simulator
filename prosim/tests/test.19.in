test2 10
DOOP 2
LOOP 3
  BLOCK 10
  DOOP 1
  LOOP 5
    BLOCK 1
    DOOP 100
  END
END
HALT

