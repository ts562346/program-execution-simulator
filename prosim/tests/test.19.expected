00000: DOOP
00002: BLOCK
00012: DOOP
00013: BLOCK
00014: DOOP
00114: BLOCK
00115: DOOP
00215: BLOCK
00216: DOOP
00316: BLOCK
00317: DOOP
00417: BLOCK
00418: DOOP
00518: BLOCK
00528: DOOP
00529: BLOCK
00530: DOOP
00630: BLOCK
00631: DOOP
00731: BLOCK
00732: DOOP
00832: BLOCK
00833: DOOP
00933: BLOCK
00934: DOOP
01034: BLOCK
01044: DOOP
01045: BLOCK
01046: DOOP
01146: BLOCK
01147: DOOP
01247: BLOCK
01248: DOOP
01348: BLOCK
01349: DOOP
01449: BLOCK
01450: DOOP
01550: HALT

DOOP count : 19
DOOP time  : 1505
BLOCK count: 18
BLOCK time : 45
